gensim/models/nmf_pgd.pyx

```diff
@@ -0,0 +1,227 @@
+# Author: Timofey Yefimov
+
+# cython: cdivision=True
```

Comment:
flag for extracting documentation from `pyx`? See other `.pyx` as example

Reply:
Ok!